packages/contracts-bedrock/justfile

```diff
@@ -59,7 +59,7 @@ snapshots-abi-storage:
 
 snapshots: build snapshots-no-build
 
-snapshots-no-build: snapshots-abi-storage kontrol-summary-fp kontrol-summary
+snapshots-no-build: semver-lock snapshots-abi-storage kontrol-summary-fp kontrol-summary
 
 snapshots-check:
```

Comment:
Also wondering if we want to keep the[ independent semver check in ci](https://github.com/ethereum-optimism/optimism/blob/58793000d7540aad4a3d3676e541cacf43de669e/.circleci/config.yml#L577-L581)?

Reply:
I think so - are you thinking we should remove it?

Reply:
Not necessarily - seems safe to keep it.  Its just that we're now running semver-lock on 2 different CI tasks and just not sure if its necessary 🤷‍♀️.  Raising it as something to consider.